pgraph/pgraph_test.go

```diff
@@ -934,3 +934,12 @@ func TestSetValue(t *testing.T) {
 		t.Errorf("expecting value of %s at %s position, got %v", value, key, v)
 	}
 }
+
+func TestGetName(t *testing.T) {
+	name := "g9"
+	G, _ := NewGraph(name)
+	graph := G.GetName()
+	if graph == "" {
```

Comment:
Is this really a meaningful check? And it's double confusing because the variable "graph" is actually the name, not the graph struct.

Reply:
Sorry for the delay. My intention was simple, 1. Create a new graph with NewGraph function which returns Graph name. 2. with the new graph structure in variable G, invokes GetName and result stored in variable "graph". 3. Finally the result is compared with the name supplied(g9).

I'm sure this is a bare minimum test which might not be useful. If it's not too much to ask. Could you direct me one how to make it more useful? 

Thanks,
Karthik

Reply:
The idea was to get you to just play with a bit of code to get new to golang. I think you will need more directed mentoring to get up to speed with golang. If you want to keep resubmitting your patches, please do, however I can't formally mentor all the basic stuff at the moment. Sorry!

Reply:
np :) I'll try to ti figure it myself and convert the code more useful.